pkgs/applications/networking/newsreaders/liferea/default.nix

```diff
@@ -6,13 +6,13 @@
 
 let
   pname = "liferea";
-  version = "1.12.4";
+  version = "1.12.6";
 in stdenv.mkDerivation rec {
-  name = "${pname}-${version}";
+  name = "${pname}-${version}b";
 
   src = fetchurl {
```

Comment:
We could change this to use fetchFromGitHub as well, then we don't have a need for the `name` attribute

Reply:
I don’t know how to use `fetchFromGitHub` to download an artifact.

Reply:
The following snippet should do it:
```nix
  src = fetchFromGitHub {
    owner = "lwindolf";
    repo = "liferea";
    rev = "v${version}";
    sha256 = "sha256:03pr1gmiv5y0i92bkhcxr8s311ll91chz19wb96jkixx32xav91d";
  };
```

Then you also should replace `fetchurl` with `fetchFromGitHub` in the inputs to the derivation.

And then it would be nice to have it all squashed together to one single commit :-)

Reply:
Note: The sha may differ because it downloads another fileformat.

Reply:
@vbgl I'm very confused about this, I tried it out myself and can't get it to build with fetchFromGitHub... So I think we can ignore that for this case...

Reply:
I can answer, the `releases/download` source used here was a manual upload using the [*releases*](https://help.github.com/articles/creating-releases/) feature from GitHub. They probably used a release-time command to generate the source tarballs and then used GitHub to host them.